src/axom/inlet/docs/sphinx/simple_types.rst

```diff
@@ -0,0 +1,144 @@
+############
+Simple Types
+############
+
+To help structure your input file, Inlet categorizes the information into
+two types: Fields and Tables.
+
+Fields refer to the individual scalar values that are either at the global level or
+that are contained inside of a Table.
+
+Tables can contain multiple Fields, other sub-Tables, as well as a single array or a single dictionary.
+
+.. note::  There is an inherent global Table that holds all global level Fields.  This can be
+  accessed via your `Inlet` class instance.
+
+
+*******
+Fields
+*******
+
+In Inlet, Fields represent an individual scalar value of primitive type.
+There are four supported field types: ``bool``, ``int``, ``double``, and ``string``.
+
+In this example we will be using the following part of an input file:
+
+.. literalinclude:: ../../examples/fields.cpp
+   :start-after: _inlet_simple_types_fields_input_start
+   :end-before: _inlet_simple_types_fields_input_end
+   :language: lua
+
+
+Defining And Storing
+--------------------
+
+This example shows how to add the four simple field types with descriptions to the
+input file schema and add their values, if present in the input file, to the Sidre
```

Comment:
I am not sure if the relationship b/w Inlet and Sidre is clear from the docs. 
It is hinted at, but not explicitly spelled out.

Perhaps a brief section or at least a sentence at the top-level stating this relationship would help?
E.g.  something like:
> Inlet stores the parsed information within a group of a Sidre Datastore.

Reply:
I added an extra paragraph to the introduction.